source/_integrations/weatherflow_cloud.markdown

```diff
@@ -17,8 +19,45 @@ ha_integration_type: integration
 
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station, you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
 
+### Which integration should I use
+
+The [WeatherFlow](https://www.home-assistant.io/integrations/weatherflow) integration provides local communications with your weather station via UDP. It returns raw sensor data readings.
+The [WeatherFlow Cloud](https://www.home-assistant.io/integrations/weatherflow_cloud) integration provides a data feed reflective of what is presented in the Tempest mobile application - including Forecast data. WeatherFlow explains the differences in their FAQ:
+
+> Can I access data from the hardware locally? To ensure access to the best data, all third-party applications and integrations should use the remote interfaces (REST & Websockets) as their primary source for data, even if they are running on the same network as the local Tempest device. There is also a local UDP interface available for those applications that require completely off-grid applications, but this should only be used as a backup to the remote interfaces.
```

Comment:
It took me a moment to understand the answer here. I would rephrase this to something like....: Yes, there is a local UDP interface...but.... To ensureaccess....
but that's my personal preference.

Reply:
Check again - made some changes.